src/lib/i18n/dictionaries/englishDictionary.ts

```diff
@@ -4150,6 +4152,9 @@ const somethingWentWrong: Record<SomethingWentWrong, string> = {
 };
 
 const specialEvent: Record<SpecialEvent, string> = {
+  "special.event.easterIntro":
```

Comment:
Should Hopper mention how many are missing? Once you find them all there is no indication that you have found them all except for going all the way back to Hopper to check. Might be worth either showing how many or indicating when you find the last one with a little 🎉